src/hotspot/share/prims/foreignGlobals.cpp

```diff
@@ -21,8 +21,8 @@
  * questions.
  */
 
-#include "foreignGlobals.hpp"
 #include "classfile/javaClasses.hpp"
+#include "foreignGlobals.hpp"
```

Comment:
This does not look right, even in the original code. Isn't it the same as `prims/foreignGlobals.hpp`, which is subsumed by the existing `prims/foreignGlobals.inline.hpp`?